main.js

```diff
@@ -26,8 +26,8 @@ class AppContainer extends React.Component {
     try {
       // add all the neccessary load in Promise.all
       let bootValues = await Promise.all([
-        loadAsync(Models.Book, "home.json"),
-        loadAsync(Models.Answer, null),
+        loadAsync(Models.Book, "home.json", true),
```

Comment:
Same question in my mind, this change will always download "home.json", how do you avoid doing that? Is there a header that we can send to server like "please don't return if your date is older than <my current cached date>"?